src/main/webapp/forms/placeGuideForm.js

```diff
@@ -7,23 +7,54 @@ function setUpCreatePlaceGuideForm() {
   addBlobstoreUploadUrlToForm(
       'CREATE_PLACE_GUIDE_FORM', 'createPlaceGuideForm');
   activatePreviewFeature();
+  activateRemoveImageFeature(false);
+  styleInputs();
 }
 
+/**
+ * This function initialises the comonents managed by the Material Design library.
+ */
+function styleInputs() {
+  const nameInput = new mdc.textField.MDCTextField(
+      document.getElementById('nameInput'));
+  const lengthInput = new mdc.textField.MDCTextField(
+      document.getElementById('lengthInput'));
+  const descriptionInput = new mdc.textField.MDCTextField(
+      document.getElementById('descriptionInput'));
+  const submitButtonRipple = new mdc.ripple.MDCRipple(
+      document.getElementById("submitBtn"));
+  const chooseAudioFileButtonRipple = new mdc.ripple.MDCRipple(
+      document.getElementById("chooseAudioFileBtn"));
+  const chooseImageFileButtonRipple = new mdc.ripple.MDCRipple(
+      document.getElementById("chooseImageFileBtn"));
+  const deletePrevImageCheckbox = new mdc.checkbox.MDCCheckbox(
+      document.getElementById('deletePrevImageCheckbox'));
+  const deletePrevImageFormField = new mdc.formField.MDCFormField(
+      document.getElementById('deletePrevImageFormField'));
+  deletePrevImageFormField.input = deletePrevImageCheckbox;
+  const publicitySwitchControl = new mdc.switchControl.MDCSwitch(
```

Comment:
We are creating the variables, but they are never used?

Reply:
Yes. Material design initializes the animations and some layout elements(for example moving the labels to up when a value is present for the textfields, character counter for the textareas, animations for the buttons and for the switches, etc.) required for these components after the initialization of the corresponding javascript objects.